modules/auth/auth_form.go

```diff
@@ -24,11 +24,13 @@ type AuthenticationForm struct {
 	AttributeName                 string
 	AttributeSurname              string
 	AttributeMail                 string
+	AttributeSSHPublicKey         string
 	AttributesInBind              bool
 	Filter                        string
 	AdminFilter                   string
 	IsActive                      bool
 	IsSyncEnabled                 bool
+	IsLdapPublicSSHKeySyncEnabled bool
```

Comment:
IsSyncEnabled and IsActive can't be checked by any other setting. Currently it is same with AdminFilter there is no special option for enabling that it is checked or not - if AdminFilter is set than Admin rights are set based on that filter, otherwise Admin rights can be set only manually.